pkgs/by-name/co/cowsql/package.nix

```
@@ -10,14 +10,15 @@
 , gitUpdater
 }:
 
-stdenv.mkDerivation rec {
+stdenv.mkDerivation {
   pname = "cowsql";
-  version = "0.15.2";
+  version = "1.15.3";
 
   src = fetchFromGitHub {
     owner = "cowsql";
     repo = "cowsql";
-    rev = "refs/tags/v${version}";
+    # this project seems to delete tags, use commit ref instead
+    rev = "a1d49d0d3e40b32ba655fffe14b7669c2aa1bcec"; # refs/tags/v1.15.3
```

```suggestion
      rev = "refs/tags/v${version}";
```

Please leave the tag rev. This is a relatively new project and it seems they decided to change their versioning scheme.